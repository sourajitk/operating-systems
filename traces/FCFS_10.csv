1,0,100
2,0,1
3,0,50
4,0,2
5,0,40
6,0,1
7,0,30
8,0,0
9,0,0
10,0,0
11,0,10
12,0,10
13,0,10
14,0,20
15,0,20
16,0,20
17,0,30
18,0,30
19,0,30
20,0,40
