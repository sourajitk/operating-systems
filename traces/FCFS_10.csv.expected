1,100
2,101
3,151
4,153
5,193
6,194
7,224
8,224
9,224
10,224
11,234
12,244
13,254
14,274
15,294
16,314
17,344
18,374
19,404
20,444
